src/main/java/org/opentripplanner/updater/stoptime/PollingStoptimeUpdater.java

```diff
@@ -136,9 +137,14 @@ public void runPolling() {
     }
   }
 
+  @Override
   public String toString() {
-    String s = (updateSource == null) ? "NONE" : updateSource.toString();
-    return "Streaming stoptime updater with update source = " + s;
+    return MoreObjects
```

Comment:
Use `ToStringBuilder`

Reply:
Done.

What are the benefits of using `ToStringBuilder`?

Reply:
I think it is the uniformity in various places, and the integrations to some OTP-specific data types.